pkgs/development/python-modules/whirlpool-sixth-sense/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, aioconsole
+, aiohttp
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "whirlpool-sixth-sense";
+  version = "0.15.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    pname = "whirlpool_sixth_sense";
+    inherit version;
+    sha256 = "0j0q80gm5g3rnqdv77azjmxkwm52ffqdlwygb1ri242qdf2hxq8z";
+  };
+
+  propagatedBuildInputs = [
+    aioconsole
+    aiohttp
+    websockets
+  ];
+
+  postPatch = ''
+    # Remove parsing of requirements.txt which doesn't exist
+    sed -i -e '6,7d' setup.py
```

Comment:
This is not really future proof.

Reply:
Addressing https://github.com/abmantis/whirlpool-sixth-sense/pull/3 would solve the issue with the parsing of `requirements.txt` but going for `unstable` works too.